docs/core/linux-prerequisites.md

```diff
@@ -94,277 +112,511 @@ To install .NET Core on RHEL 7:
-     dotnet --help
- ```
-## Install .NET Core for Debian 8 or 9 (64 bit).
+ ## Install .NET Core for Debian 8 or Debian 9 (64 bit)
 
-> [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+To install .NET Core on Debian 8 or Debian 9 (64 bit):
 
-To install .NET Core on Debian 8 or 9 (64 bit):
-1. Get the prerequisites. 
-    ```bash
-    sudo apt-get install curl libunwind8 gettext
-    ```
-2. Download the .NET Core SDK binaries (tarball).
+1. Remove any **previous preview** versions of .NET Core from your system.
 
-# [.NET Core 2.x](#tab/netcore2x)   
+> [!NOTE]
+> A user-controlled directory is required for Linux system installs from tar.gz.
+
+# [.NET Core 2.x](#tab/netcore2x)
+
+2. Install system components.
+
+   ```bash
+   sudo apt-get update
+   sudo apt-get install curl libunwind8 gettext apt-transport-https
+   ```
+   
+3. Register the trusted Microsoft Product key.
+
+   ```bash
+   curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+   sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+   ```
+   
+4. Register the Microsoft Product feed.
+
+   **Debian 9 (Stretch)**
+
+   ```bash
+   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" > /etc/apt/sources.list.d/dotnetdev.list'
+   ```
+   
+   **Debian 8 (Jessie)**
+   
+   ```bash
+   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-jessie-prod jessie main" > /etc/apt/sources.list.d/dotnetdev.list'
+   ```
+   
+5. Extract the .NET Core SDK binaries.
+
+   ```bash
+   sudo apt-get update
+   sudo apt-get install dotnet-sdk-2.0.0
+   ```
+
+6. Add dotnet to your PATH.
+
+   ```bash
+   export PATH=$PATH:$HOME/dotnet
+   ```
+
+# [.NET Core 1.x](#tab/netcore1x)
+
+2. Get the prerequisites.
+
+   ```bash
+   sudo apt-get install curl libunwind8 gettext
+   ```
+
+3. Download the .NET Core SDK binaries (tarball).
+
+   ```bash
+   curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848826
+   ```
+
+4. Extract the .NET Core SDK binaries.
 
-```bash
-     curl -sSL -o dotnet.tar.gz https://aka.ms/dotnet-sdk-2.0.0-linux-x64
-```
+   ```bash
+   sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
+   ```
 
-# [.NET Core 1.x](#tab/netcore1x)   
+5. Add dotnet to your PATH.
+
+   ```bash
+   sudo ln -s /opt/dotnet/dotnet /usr/local/bin
+   ```
 
-```bash
-     curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848826
-```
 ---
-3. Extract the .NET Core SDK binaries.
-    ```bash
-    sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
-    ```
-4. Add dotnet to your PATH.
-    ```bash
-    sudo ln -s /opt/dotnet/dotnet /usr/local/bin
-    ```
-5. Run the `dotnet --help` command to prove the installation succeeded.
 
-     ```bash
-     dotnet --help
-     ```
+6. Run the `dotnet --version` command to prove the installation succeeded.
```

Comment:
consider adding this inside the tab since numbering is different between one tab and another? otherwise for 2.x you have steps number 6

Reply:
Thank you @mairaw!